src/main/java/com/minecolonies/network/messages/SaveScanMessage.java

```diff
@@ -0,0 +1,66 @@
+package com.minecolonies.network.messages;
+
+import com.minecolonies.items.ItemScanTool;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.items.ItemScanTool'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)